sharepoint/sharepoint-ps/sharepoint-online/Get-SPOEnterpriseAppInsightsReport.md

```diff
@@ -0,0 +1,118 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: 
+applicable: SharePoint Online
+title: Get-SPOEnterpriseAppInsightsReport
+schema: 2.0.0
+author: sumikumar
+ms.author: sumikumar
+ms.reviewer:
+manager: hikakar
+---
+
+# Get-SPOEnterpriseAppInsightsReport
+
+## SYNOPSIS
+
+This cmdlet enables the admin to check status of all active and available reports when no report ID is present and to View/Download a Report if report ID is present.
+
+## SYNTAX
+
+```powershell
+Get-SPOEnterpriseAppInsightsReport [-ReportId <Guid>] [-Action <enum>]
+```
+
+## DESCRIPTION
+
+If this cmdlet is executed without any parameters, it displays the status of all active and completed reports (adhering to any retention timeline as per DAG) with the following properties:
+| Property             | Description                              |
+| :------------------- | :--------------------------------------- |
+| Id | The unique Id of the report.                    |
+| CreatedDateTimeInUtc | The Date and Time the report creation was triggered in UTC.                   |
+| Status | The Status of the report.               |
+| ReportPeriodInDays | The report duration in days.       |
```

Comment:
please align this markdown table. if you edit this with VS Code it should auto-format to align it for you